src/extension.ts

```diff
@@ -16,7 +16,7 @@
 
 import * as vscode from 'vscode';
 
-import {CodelensProvider} from './Codelens/CodelensProvider';
+import {CodelensProvider} from './Editor/CodelensProvider';
```

Comment:
Currently there is no `Editor` folder.

Reply:
There is ` src/Codelens/CodelensProvider.ts → src/Editor/CodelensProvider.ts ` in this PR.
I've missed this.